### DISCUSSION TITLE
[Question][Hw1] How can we submit and check it?

### DISCUSSION THREAD

**Discussion by Allgot** [prosyslab-classroom/cs424-program-reasoning#390] [Q&A]
I'm new to this github-classroom, so I'm not sure whether I submitted successfully.

Is it successfully submitted if I can see the green check marker in the action tap of my homework repository? And is it intentional that I cannot check any score? (or is it bug?)

Thank you!

**Comment by sujin0529** [comment]
If you see the green check marker, it means that you have submitted homework that works fine for two examples.
And grading is done through an undisclosed test case.
So it's not a bug that you can't check your score now!

**Comment by Allgot** [comment]
I got it!
Thank you :)
